client/extensions/woocommerce/state/ui/shipping/zones/methods/flat-rate/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	WOOCOMMERCE_SHIPPING_ZONE_METHOD_SET_TAXABLE,
+	WOOCOMMERCE_SHIPPING_ZONE_METHOD_SET_COST,
+} from 'woocommerce/state/action-types';
+
+const initialState = {
+	taxable: 'None',
+	cost: 0,
+};
+
+const reducer = {};
+
+reducer[ WOOCOMMERCE_SHIPPING_ZONE_METHOD_SET_TAXABLE ] = ( state, { isTaxable } ) => {
+	return { ...state,
+		taxable: isTaxable ? 'Taxable' : 'None',
```

Comment:
My bad, I always intended to match server-side names. Fixed in https://github.com/Automattic/wp-calypso/pull/14890/commits/5ac6a1e9ba46df54a67443bc7990f088c681a9ff